client/settings/general-settings-section/payment-method-description.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import styled from '@emotion/styled';
+
+const Wrapper = styled.div`
```

Comment:
Moved some of these components from `client/settings/general-settings-section/index.js`